RELEASE_NOTES.rst

```diff
@@ -4,6 +4,7 @@ Next release
 All changes
 -----------
 
+- :pull:`456`: Fix bug in :func:`.years_active` to use the lifetime corresponding to the year_vintage for which the active years are being retrieved.
```

Comment:
This should probably be :meth: rather than :func:. Does the link work when the docs are built?

Reply:
@khaeru - changes made; link works.